frameworks/sveltekit/poc/cjs.cjs

```diff
@@ -0,0 +1,16 @@
+Object.defineProperty(exports, "__esModule", { value: true });
+
+const Prisma = {};
+exports.Prisma = Prisma;
+
+Prisma.prismaVersion = {
+  client: "3.12.0",
+  engine: "22b822189f46ef0dc5c5b503368d1bee01213980"
+};
```

Comment:
What is this about @tnzk? This already is outdated and will get more and more outdated. Do we really need it?

Reply:
No, the entire `poc` directory is nothing but a concept proof of this fix just for me. Will remove it later.